week1/3-website/index.html

```diff
@@ -18,5 +18,178 @@
+                <ol type="a">
+                  <li>
+                    <b>The Narrator:</b><br>
+                       - Unlike Sa’eed he is “a being with a background, with roots, with a purpose.”
+                       Returning to the Village after seven years of studies in London, he feels integrated
+                       within the family, his heart flowing with love, his feeling of security being
+                       strengthened.<br>
+                       - Meeting Sa’eed, will he be infected with “germs” Sa’eed carries in his system?
+                       The ambiguity of the relationship: Is Sa’eed narrator’s double, his foil?<br>
+                       - Will Sa’eed “Kill” the young man?<br>
+                       - No, the Narrator refuses to die, He calls his daughter “Hope” and looks forward
+                       to the future.<br>
+                       - He returns to tell us his story in the light of Sa’eed’s. It is the story of the
+                       educated Arab man in the post-colonial epoch, of the civil servant, fully aware of
+                       the corruption of his rulers, his only non-passive gesture is to opt out for life. Will he
+                       find his place in “the scheme of things”? Will he be able to help his country? His
+                       personality is in the shaping, he bears no name yet.
+                  </li>
+                  <li>
+                    <b>Mahjoub:</b><br> What the narrator would have been had he not left the village.
+                       Intelligent, powerful, respected; yet he is frustrated by rural life and dreams of city
+                       life. A realist, he accepts the old mentality of the village and does not sympathize
+                       with Hosna nor with the narrator’s breakdown.
+                  </li>
+                  <li>
+                    <b>Hosna:</b><br> The playmate of the Narrator and Mahjoub when they were kids, she
+                       changed when she married Sa’eed becoming “like a city woman”.<br>- A stranger in her own village?<br>
+                       - Her fatal end.<br>
+                       - She is the only tragic figure in the novel. In dignity and with her own blood, she
+                       announces the birth of the new Arab woman.
+                  </li>
+                </ol>
+              </li>
+            </ol>
+              You can buy the book from <a href="https://www.amazon.co.uk/Tayeb-Salih-Migration-Penguin-Classics/dp/B00I61GOYG/ref=pd_lpo_14_img_0/257-9467674-1352042?_encoding=UTF8&pd_rd_i=B00I61GOYG&pd_rd_r=9ea6a7b8-ed94-42dd-a3e4-520b9a06f3f4&pd_rd_w=gJ6tR&pd_rd_wg=qx9iu&pf_rd_p=7b8e3b03-1439-4489-abd4-4a138cf4eca6&pf_rd_r=E0VAZPPVPP27Q4CFHG1N&psc=1&refRID=E0VAZPPVPP27Q4CFHG1N" target="_blank"><em>Amazon</em></a>.
+          </p>
+        </article>
+      </section>
+      <footer id="footer">
+      ©  All copyrights and disclaimers are reserved. 
```

Comment:
I like the thought behind this, but if you aren't quoting sources, it isn't very meaningful.